.github/workflows/node-release.yml

```diff
@@ -0,0 +1,104 @@
+name: Node Release
+
+env:
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  AWS_DEFAULT_REGION: us-east-1
+
+on:
+  push:
+    tags:
+      - node-v*
+
+jobs:
+  build_all:
```

Comment:
Is this step doing something that the next one doesn't? Otherwise, since the `npm_publish` depends on `rust_publish` we can probably remove it. If one of the builds (in `rust_publish`) for any of the os/node were to fail, it would just fail the whole job anyway, right?
I think it would maybe make more sense to run the `build_all` job for each commit, to make sure we don't break anything in these bindings. What do you think?

Reply:
`rust_build` only builds rust, not typescript so that's why. I agree it would be better to run that at each commit, but maybe only the ts build then (and only when files inside `bindings/node` are updated or something like that)? There is already a rust build running at each commit.

Reply:
And js tests could be integrated at this step too (when they arrive 😆)

Reply:
Sure but the `Rust` job doesn't check that the bindings still compile after something has been modified in the main library. At each commit, I think we would like to make sure everything is still compiling: the main library, but also all the bindings.

Reply:
yes you're right, I pushed a new workflow `node.yml` with the `build_all` job and removed it from the release workflow

Reply:
And we can maybe add a `paths-ignore` with `./bindings` in `rust.yml` since I don't see the need to rebuild the rust source if only code in bindings has been updated?

Reply:
Nice thanks! You are right, at the moment it's totally useless. I don't know what would be better between having a separate workflow for the main library and each bindings, or actually having one main `Build` workflow, that takes care of building everything on each OS, mutualizing the setup parts. What do you think? (This is out of the scope of this PR though).